Kinder/Classes/User.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+public class User
+{
+    public string Name { get; set;}
+    public string SurName { get; set; }
+    public string Username { get; set; }
+    public string Email { get; set; }
+    public string PhoneNumber { get; set; }
+    public int Id { get; set; }
+    public int KarmaPoints { get; set; }
```

Comment:
Getters and setters on public variables that should be private since it is user information and not all of them should have both a getter and setter.

Reply:
Compiler sees differently. Here sending a link so you could take a look at how actually compiler sees code. https://codeasy.net/lesson/properties

Reply:
I was not aware of this feature of C#, my mistake, however do we still want setters on things like Username and Id when their value should be generated within the same class? There's also no password variable for testing purposes.

Reply:
Not sure about possibility to change username, but with all other notices i would agree. So i'll add password variable and delete setter for ID.